Move Request Icon to Title Bar in Detail Screen

EHaake opened the pull request:
Create custom IconDisplay component and add it to the title bar.
Remove from the Detail Screen itself because there was an annoying
formatting issue when the title was split into multiple lines

EHaake committed:
Move Request Icon to Title Bar in Detail Screen

Create custom IconDisplay component and add it to the title bar.
Remove from the Detail Screen itself because there was an annoying
formatting issue when the title was split into multiple lines

SimonBartonPSU commented:
Looks like this overlapped on iOS. I say we still merge though and this fix can be a separate issue.

![IMG-1642](https://user-images.githubusercontent.com/46763450/86497041-d81dc480-bd34-11ea-81cd-2e3d0f3a8dfa.jpg)

SimonBartonPSU reviewed (dismissed):
Just realized I left comment, not review. :+1:

EHaake committed:
Fix Custom Title Rendering

Make custom back button text for iOS which just prints 'Back' so there
shouldn't be any overlap.

Conditional styling to fix apparent margin difference between the two icons

Tested on iOS and Android

EHaake commented:
iOS screenshots for proof.
![image1](https://user-images.githubusercontent.com/20587371/86502701-e252b980-bd5a-11ea-81f6-359286169ec2.png)
![image0](https://user-images.githubusercontent.com/20587371/86502703-e2eb5000-bd5a-11ea-931c-06bf939c7908.png)

EHaake committed:
Merge branch 'master' into erik-move-icons-to-titleBar
